bin/obj2gltf.js

```diff
@@ -128,6 +128,16 @@ if (defined(argv.metallicRoughnessOcclusionTexture) && defined(argv.specularGlos
 var objPath = argv.input;
 var gltfPath = argv.output;
 
+if (objPath === '.') {
+    console.log('Input path is required');
+    return;
+}
+
+if (gltfPath === '.') {
+    console.log('Output path must be a filename');
+    return;
+}
```

Comment:
Can we embed this into `yargs` like https://github.com/AnalyticalGraphicsInc/agi-swayze/blob/swayzedb-textures/bin/roadHouse.js#L29-L50?

Reply:
Updated to use `coerce`.

I also tried `check` but the error message wasn't as helpful ultimately.